code/modules/clothing/under/shorts.dm

```diff
@@ -1,279 +1,144 @@
+//
+// Shorts
+//
+
+// Shorts Parent Item
 /obj/item/clothing/under/shorts
-	name = "athletic shorts"
-	desc = "95% Polyester, 5% Spandex!"
```

Comment:
This should be the actual desc instead of DESC_PARENT, unless you want to manually copy paste this description for every single child. If you want something more general to encompass all shorts, write something else. Reuse "95% Polyester, 5% Spandex!" for the athletic shorts parent desc.

DESC_PARENT is an interesting idea. However, the actual fact is that in descriptions, you *WANT* the parent description to be passed down. You should put the "You shouldn't be seeing this" in the *name* instead, since it's more likely to be changed for the children than passed down.
Get rid of DESC_PARENT. It's going to set a bad precedent of manually setting descriptions for each and every child, which is something not good.